frontend/src/concepts/pipelines/content/ViewPipelineServerModal.tsx

```diff
@@ -48,62 +48,61 @@ const ViewPipelineServerModal: React.FC<ViewPipelineServerModalProps> = ({
     >
       {pipelineNamespaceCR && (
         <DescriptionList termWidth="20ch" isHorizontal>
-          {!!pipelineNamespaceCR.spec.objectStorage &&
-            !!pipelineNamespaceCR.spec.objectStorage.externalStorage &&
+          {!!pipelineNamespaceCR.spec.objectStorage.externalStorage &&
             !!pipelineNamespaceCR.spec.objectStorage.externalStorage.s3CredentialsSecret
```

Comment:
Can this be simplified to:
```suggestion
          {!!pipelineNamespaceCR.spec.objectStorage.externalStorage?..s3CredentialsSecret
```